common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1130,33 +1131,33 @@
+    "figura.docs.model_part.get_rot": "Gets the rotation of the model part, including its rotation in BlockBench\nFor relative rotation values, check out the \"offset\" rot functions",
     "figura.docs.model_part.set_rot": "Sets the absolute rotation for this part\nNil values for rotation are assumed to be 0\nAngles are given in degrees\nFor relative rotation values, check out the \"offset\" rot functions",
-    "figura.docs.model_part.get_offset_rot": "Gets the offset rotation of the model part, offset from its rotation in blockbench\nFor absolute rotation values, check out the non-offset rot functions",
+    "figura.docs.model_part.get_offset_rot": "Gets the offset rotation of the model part, offset from its rotation in BlockBench\nFor absolute rotation values, check out the non-offset rot functions",
     "figura.docs.model_part.set_offset_rot": "Sets the offset rotation for this part\nNil values for rotation are assumed to be 0\nAngles are given in degrees\nFor absolute rotation values, check out the non-offset rot functions",
     "figura.docs.model_part.get_anim_rot": "Gets the rotation offset provided by the currently active animation of this model part",
     "figura.docs.model_part.get_true_rot": "Gets the true rotation of this model part, which is a sum of the rotation, the offset rotation and the animation position",
-    "figura.docs.model_part.get_scale": "Gets the scale of the model part, as a multiple of its blockbench size\nOnly changes from {1,1,1} when you call setScale()",
+    "figura.docs.model_part.get_scale": "Gets the scale of the model part, as a multiple of its BlockBench size\nOnly changes from {1,1,1} when you call setScale()",
     "figura.docs.model_part.set_scale": "Sets the scale factor for this part\nNil values for scale are assumed to be 1",
     "figura.docs.model_part.get_offset_scale:": "Gets the offset scale of the model part, offset from its default scale\nFor absolute scale values, check out the non-offset rot functions",
     "figura.docs.model_part.set_offset_scale:": "Sets the offset scale for this part\nNil values are assumed to be 1\nFor absolute scale values, check out the non-offset rot functions",
     "figura.docs.model_part.get_anim_scale": "Gets the scale multiplier provided by the currently active animation of this model part",
     "figura.docs.model_part.get_true_scale": "Gets the true scale of this model part, which is a sum of the scale, the offset scale and the animation scale",
-    "figura.docs.model_part.get_pivot": "Gets the pivot point of the model part, including its pivot in blockbench\nFor relative values, check out the \"offset\" pivot functions",
+    "figura.docs.model_part.get_pivot": "Gets the pivot point of the model part, including its pivot in BlockBench\nFor relative values, check out the \"offset\" pivot functions",
```

Is there a way to put a comment on multiple separate lines? It seems to only allow me to select ranges.
```suggestion
    "figura.docs.model_part.get_pivot": "Gets the pivot point of the model part, including its pivot in Blockbench\nFor relative values, check out the \"offset\" pivot functions",
```